qontract_api/qontract_api/integrations/sso_client/tasks.py

```
@@ -0,0 +1,119 @@
+    self: Any,  # Celery Task instance (bind=True)
+    ocm_environment: str,
+    clusters: list[SsoClientCluster],
+    keycloak_secrets: list[KeycloakInstanceSecret],
+    vault_target: Secret,
+    *,
+    dry_run: bool = True,
+) -> SsoClientTaskResult:
+    """Reconcile RHIDP SSO clients (background task)."""
+    request_id = self.request.id
+
+    try:
```

Ruff (0.15.21)

[warning] Try clause contains too many statements (9 > 5)

(PLW0717)